lib/tasks/statcan.rake

```diff
@@ -0,0 +1,66 @@
+namespace :statcan do
+  desc "Setup Statcan datasets"
+  task setup_datasets: :environment do
+    statcan_datasets = [
```

Comment:
Names/urls/schedules taken from the existing workflows https://github.com/BuildCanada/OutcomeTracker/tree/c165db79919c77fc66f0663c5267bd0b0e300337/.github/workflows.

Is there is a specific reason for the existing schedules?

Reply:
Note, we might not need this task now that the Avo resource has been added, as it's easy to add the datasets via the admin panel.

Reply:
It'd be good to add this to the db seeds instead so people have a good database to start with `db/seeds/canada.rb